GitCommands/Git/GitModule.cs

```diff
@@ -2464,7 +2528,12 @@ public string GetStatusText(bool untracked)
         public string GetDiffFilesText(string firstRevision, string secondRevision, bool noCache = false)
         {
             string cmd = DiffCommandWithStandardArgs + "-M -C --name-status " + _revisionDiffProvider.Get(firstRevision, secondRevision);
-            return noCache ? RunGitCmd(cmd) : RunCacheableGitCmd(cmd, SystemEncoding);
+            KeyValuePair<string, string>[] configs = new KeyValuePair<string, string>[]
```

Comment:
Can this be broken out so this is still centralized.
Now the config is repeated four times.